auto_dev/data/templates/protocols/primitive_strategies.jinja

```diff
@@ -0,0 +1,28 @@
+"""Module containing hypothesis strategies for the custom primitives."""
```

Comment:
these `primitives` and hence `primitive_strategies` are not protocol dependent. They still get generated for all protocols, since there is no easy way to provide them as a base module, but they are used in several more than one of the protocol test files and this cleans up the clutter from protocol-specific generated test code as well.